### ISSUE TITLE
Automate the selection of black or white NPMaki icons based on color selection

### ISSUE THREAD

**Issue by jakecoolidge** [nationalparkservice/npmap-builder#206]
We've discussed shortening the NPMaki drop-down menu in Builder and eliminating color contrast issues (for example, black icons placed on black place markers) by automating the black/white icon selection based on the selected color.

**Comment by nateirwin** [comment]
This is ready to go in master. Let’s walk through and deploy this afternoon.
